content/en/docs/ibm/pipelines.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Pipelines on IBM Cloud Kubernetes"
+description = "Instructions for using Kubeflow Pipelines on IBM Cloud Kubernetes"
+weight = 50
++++
+
+## Authenticating Pipelines with the SDK
+
+**Notes**:
+* Python package `kfp` v1.0.0 is required.
+* This feature is available with multi-user, auth-enabled Kubeflow installation deployed from the manifest [https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml).
+* Since it transports sensitive info like session cookie value over edge network, it highly recommends enabling HTTPS for the public endpoint of Kubeflow.
+
+It requires authentication via the public endpoint of Kubeflow deployment when using the Kubeflow Pipelines multi-user feature with Pipelines SDK. It will need to provide below variables no matter from an in-cluster Jupyter notebook or remote machine:
+1. `KUBEFLOW_PUBLIC_ENDPOINT_URL` - Kubeflow public endpoint URL. You can obtain it from command `ibmcloud ks nlb-dns ls --cluster <your-cluster-name>`.
+1. `SESSION_COOKIE` - A session cookie starts with `authservice_session=`. You can obtain it from your browser after authenticated from Kubeflow UI.
```

Comment:
Although this is a limitation for DEX and KFP, maybe add a note the session will timeout after certain timeframe. Otherwise it looks good to me.